packages/max31865-1.0.0.yaml

```diff
@@ -2,4 +2,4 @@ name: max31865
 description: Driver for MAX31865 RTD-to-Digital Converter, an SPI-connected digital-to-analog converter, typically used for temperature measurement.
 license: MIT
 url: github.com/toitware/max31865-driver
-version: 1.0.0
+version: 1.0.1
```

Comment:
Needs the dependencies.

Reply:
I added the direct dependency, but not the transitive one.  Hope this is correct.

Reply:
That is correct.